src/components/prometheusmetrics.ts

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 
 import PromClient, { Registry } from "prom-client";
 import { AgeCounters } from "./agecounters";
-import MatrixClient from "matrix-js-sdk";
+import JsSdk from "matrix-js-sdk";
```

Comment:
Can you `{ MatrixClient }` if you don't need other stuff from the Sdk?

Reply:
Sadly not, as it's not exported as a type. You can access it, but TS will complain.